jicofo/src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -654,7 +663,9 @@ private void onMemberJoined(@NotNull ChatRoomMember chatRoomMember)
                             + " videoMuted=" + chatRoomMember.isVideoMuted()
                             + " role=" + chatRoomMember.getRole()
                             + " isJibri=" + chatRoomMember.isJibri()
-                            + " isJigasi=" + chatRoomMember.isJigasi());
+                            + " isJigasi=" + chatRoomMember.isJigasi()
+                            + " isTranscriber=" + chatRoomMember.isTranscriber()
+                            + room);
```

Comment:
Isn't this already in the logger metadata?

Reply:
The JID of the main room is, I'd like to log which visitor node it's on (if it's a visitor)